cpio/writerhelper.go

```diff
@@ -109,7 +109,7 @@ func (w *WriterHelper) CopyTree(path string) {
 		} else if info.Mode().IsRegular() {
 			w.CopyFile(p)
 		} else {
-			panic("No handled")
+			log.Printf("Ignored link: %s/%s", path, p);
```

Comment:
Stopping to panic is good; ignoring and moving forward is not.. Can you explain in your commit mesage what you're trying to solve here and why in that case it's reasonably to not copy?

Reply:
There used to by symbolic links to some open gl configuration in the ubuntu /etc.
I can't find them now, so it may no longer be required